http/src/main/java/io/micronaut/http/form/FormConfigurationProperties.java

```diff
@@ -45,11 +53,27 @@ public int getMaxDecodedKeyValueParameters() {
         return maxDecodedKeyValueParameters;
     }
 
+
+    /**
+     * @return true if the semicolon is treated as a normal character, false otherwise
+     */
+    @Override
+    public boolean isSemicolonIsNormalChar() {
+        return semicolonIsNormalChar;
+    }
+
     /**
      * default maximum of decoded key value parameters. Default value {@link #DEFAULT_MAX_DECODED_KEY_VALUE_PARAMETERS}.
      * @param maxDecodedKeyValueParameters default maximum of decoded key value parameters
      */
     public void setMaxDecodedKeyValueParameters(int maxDecodedKeyValueParameters) {
         this.maxDecodedKeyValueParameters = maxDecodedKeyValueParameters;
     }
+
+    /**
+     * @param semicolonIsNormalChar true if the semicolon should be treated as a normal character, false otherwise
+     */
+    public void setSemicolonIsNormalChar(boolean semicolonIsNormalChar) {
+        this.semicolonIsNormalChar = semicolonIsNormalChar;
+    }
```

Comment:
I don't see any changes to `DefaultFormUrlEncodedDecoder` that would respect this configuration

Reply:
fixed